src/fun/counting/main.go.tmpl

```diff
@@ -0,0 +1,91 @@
+{{/*
+        Counting command. Count and have fun!
+        
+        Author: H1nr1 <https://github.com/H1nr1>
+*/}}
+
+{{/* Configurable Values */}}
+{{ $Admin := 123456789 }} {{/* Administrator role ID; EDIT THIS TO A ROLE ID YOU HAVE */}}
+{{/* Optional */}}
+{{ $CountTwice := true }} {{/* Allow users to count multiple times in a row; true/false */}}
+{{ $RoleID := false }} {{/* Counting role ID; set to false to disable */}}
+{{ $SecondChance := true }} {{/* Second chance if wrong; true/false */}}
+{{ $Leaderboard := true }} {{/* If you added the leaderboard CC; true/false */}}
+{{ $CorrectEmoji := "✅" }} {{/* Emoji to react with if number is correct; if custom, use format name:id */}}
+{{ $WarningEmoji := "⚠️" }} {{/* Emoji to react with if wrong number AND Second Chance is true/on; if custom, use format name:id */}}
+{{ $IncorrectEmoji := "❌" }} {{/* Emoji to react with if number is incorrect; if custom, use format name:id */}}
+{{/* End of configuration values */}}
+
+{{/* No Touchy */}}
+{{ with .ExecData }}
+	{{ if not (getMessage (toInt (dbGet 0 "Channel").Value) .) }} {{/* Check if number was deleted */}}
+		{{ sendMessage (toInt (dbGet 0 "Channel").Value) (cembed 
+			"description" (print "<@" (dbGet 0 "LastUser").Value "> deleted their number which was correct!\nThe next number is " (dbGet 0 "Next").Value)
+			"color" 30654) }}
+	{{ return }}{{ end }}
+{{ end }}
+
+{{/* Initializing variables */}}
+{{ $LastUser := (dbGet 0 "LastUser").Value }}{{ $Next := (dbGet 0 "Next").Value }}
+{{ $Number := "" }}{{ $Number = round (slice ($Number = (exec "calc" (index .Args 0))) 9 (len $Number | add -1)) }}
```

Comment:
As already mentioned, this is silly. The entire line can be replaced with `{{ $Number := toInt .Message.Content }}`

Reply:
This is not correct, the documentation mentions that it also accepts mathematical expressions, which aren't converted with `toString`.

I suggest to either revise this line to make it more readable, or to outright remove this feature.

Reply:
Sorry, I meant in this particular context when (as far as I understand) mathematical expressions wouldn't be used, only numbers. Maybe I'm misunderstanding the context?

Reply:
See https://github.com/yagpdb-cc/yagpdb-cc/blob/c5662bc24501c2d7d10268a622422a272e771c93/website/docs/fun/counting/overview.md?plain=1#L58

Reply:
Skimmed over that, thank you : )